pulser-core/pulser/backend/remote.py

```diff
@@ -148,6 +157,7 @@ def __init__(
                 "'connection' must be a valid RemoteConnection instance."
             )
         self._connection = connection
+        self._batch_id = ""
```

Comment:
nit: I think the default should be None to be consistent with the submit method args

Reply:
I agree

Reply:
Mypy thinks otherwise